### PR TITLE
Update Makefile for proper integration of git hooks script

### PR DESCRIPTION
**Pull request by davidferlay**
### Observed result

- Newly added Makefile instructions are executed for every `make ..` command
```
$(info Installing git hooks)
$(shell ln -sfn ../../scripts/git_hooks/pre-commit.sh .git/hooks/pre-commit)
```

### Expected result

- Creation of script symbolic link should not happen at every `make ..` command
- A new `make ..` command could be useful for developer to run script (like it's possible for phpcs for ex)

### Todo

- Move symlink creation to `make hooksymlink``
- Add `make hooksymlink` to `make all`
- Create another make command `make hookexec` to just run git hook script itself

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -150,3 +148,14 @@ phpcs:
 ## Fix codebase according to Drupal standards https://www.drupal.org/docs/develop/standards
 phpcbf:
 	@$(call phpcsexec, phpcbf)
+
+## Add symbolic link from custom script(s) to /.git/hooks/pre-commit
+hooksymlink:
```
Comment (andypost) [inline Makefile]: This needs to point a file to check

Comment (davidferlay) [reply Makefile]: Updated

Comment (andypost) [reply Makefile]: Follow up should consider checking for target using make https://stackoverflow.com/questions/35588153/where-does-make-store-its-cache/35590775#35590775

**Comment by jbguerraz** [comment]
follow-up https://github.com/skilld-labs/skilld-docker-container/issues/70

**Pull request merged**